src/classes/HH_Households_TEST.cls

```diff
@@ -575,44 +575,56 @@ public with sharing class HH_Households_TEST {
     }
     
     /*********************************************************************************************************
-    * @description Tests merging a contact into a Household updates rollups on the Household
+    * @description Tests merging a contact into a Household updates rollups on the Household when usin gthe
+    * One-to-One Account Model
     */
-    static testMethod void testMergeRollupSupport(){
-        npe01__Contacts_and_Orgs_Settings__c contactSettingsForTests = UTIL_CustomSettingsFacade.getContactsSettingsForTests(new npe01__Contacts_and_Orgs_Settings__c (npe01__Account_Processor__c = CAO_Constants.ONE_TO_ONE_PROCESSOR, npe01__Enable_Opportunity_Contact_Role_Trigger__c = true));
-        npo02__Households_Settings__c householdSettingsForTests = UTIL_CustomSettingsFacade.getHouseholdsSettingsForTests(new npo02__Households_Settings__c (npo02__Household_Rules__c = HH_Households.ALL_PROCESSOR, npo02__Advanced_Household_Naming__c = true, npo02__Enable_Opp_Rollup_Triggers__c = false));
-         
+    /*static testMethod void testMergeRollupSupport(){
```

Comment:
Let's delete the commented out unit test from the class.

Reply:
Thanks. I forgot this has been commented out in my initial proof of concept. Might go back to enable this with some other related changes.